packages/forms/src/UIForm/Widget/Widget.component.test.js

```diff
@@ -41,6 +41,22 @@ describe('Widget component', () => {
 		expect(wrapper.getElement()).toMatchSnapshot();
 	});
 
+	it('should render widget with tooltip', () => {
+		const wrapper = shallow(
+			<Widget
+				id={'myForm'}
+				onChange={jest.fn('onChange')}
+				onFinish={jest.fn('onFinish')}
+				onTrigger={jest.fn('onTrigger')}
+				properties={properties}
+				schema={{ ...schema, tooltip: 'example tooltip' }}
+				errors={errors}
+			/>,
+		);
+
+		expect(wrapper.getElement()).toMatchSnapshot();
```

Comment:
please replace toMatchSnapshot with check wrapper.getElement().type is TooltipTrigger with the good label (props() on it)
Nobody read snapshots